index.html

```diff
@@ -179,16 +196,22 @@ <h1 class="title">BitcoinJS</h1>
 
         <h2>Creating a Transaction</h2>
         <pre class="prettyprint">
+
 var tx = new bitcoin.TransactionBuilder()
 
-// Add the input (who is paying) of the form [previous transaction hash, index of the output to use]
-tx.addInput("aa94ab02c182214f090e99a0d57021caffd0f195a81c24602b1028b130b63e31", 0)
+// Add the input (who is paying):
+// [previous transaction hash, index of the output to use]
+
+var txHash = 'aa94ab02c182214f090e99a0d57021caffd0f195a81c24602b1028b130b63e31'
```

Comment:
Can this be named `txId`?
As a convention, we've used `txHash` for the regular-byte-order `Buffer` of the `SHA256(SHA256(tx))`, and `txId` for the reverse-byte-order equivalent.